de_web_plugin.cpp

```diff
@@ -325,10 +325,31 @@ DeRestPluginPrivate::DeRestPluginPrivate(QObject *parent) :
     }
 
     // create default group
-    Group group;
-    group.setAddress(0);
-    group.setName("All");
-    groups.push_back(group);
+    if (gwGroup0 == 0) { // get new id and replace old group0 and get new id
```

Comment:
Please always use braces on the next line for if, for. So that the code base just has one code style.
```
if (gwGroup0)
{
}
```

Reply:
Thanks, missed this one, it's no my usual code style.